apps/analysis/mutation.py

```diff
@@ -289,3 +337,8 @@ class Mutation():
     # -- Uploads
     analysis_report_upload_create = CreateAnalysisReportUpload.Field()
     analysis_report_upload_delete = DeleteAnalysisReportUpload.Field()
+    # Analysis
+    analysis_create = CreateAnalysis.Field()
+    analysis_update = UpdateAnalysis.Field()
+    analysis_delete = DeleteAnalysis.Field()
+    analysis_bulk = BulkAnalysis.Field()
```

Comment:
Can we add detail for this mutation in a comment. I mean what bulk operations it can do. example create, update , delete...